supervisor/addons/build.py

```diff
@@ -121,39 +122,74 @@ def build_is_valid() -> bool:
         except HassioArchNotFound:
             return False
 
-    def get_docker_args(self, version: AwesomeVersion, image: str | None = None):
-        """Create a dict with Docker build arguments.
+    def get_addon_host_path(self) -> str:
+        """Get the host path for the addon directory.
 
         Must be run in executor.
         """
-        args: dict[str, Any] = {
-            "path": str(self.addon.path_location),
-            "tag": f"{image or self.addon.image}:{version!s}",
-            "dockerfile": str(self.get_dockerfile()),
-            "pull": True,
-            "forcerm": not self.sys_dev,
-            "squash": self.squash,
-            "platform": MAP_ARCH[self.arch],
-            "labels": {
-                "io.hass.version": version,
-                "io.hass.arch": self.arch,
-                "io.hass.type": META_ADDON,
-                "io.hass.name": self._fix_label("name"),
-                "io.hass.description": self._fix_label("description"),
-                **self.additional_labels,
-            },
-            "buildargs": {
-                "BUILD_FROM": self.base_image,
-                "BUILD_VERSION": version,
-                "BUILD_ARCH": self.sys_arch.default,
-                **self.additional_args,
-            },
+        # The addon path will be mounted from the host system
+        relative_path = self.addon.path_location.relative_to(
+            self.sys_config.path_supervisor
+        )
+        return str(self.sys_config.path_extern_supervisor / relative_path)
```

Comment:
There is a helper to do that, search for `local_to_extern_path`.

Reply:
Awesome, this allowed me to simplify stuff.